### PR TITLE
Heroku professional 99440996

### PR DESCRIPTION
**Pull request by variousred**
turns on the professional tier after app creation so we can use standard dynos

### CODE REVIEW

**Comment by variousred** [comment]
@levibrown @thescubageek @quady

File: lib/client_app_provision_modules.rb [hunk 1]
```diff
@@ -64,9 +64,19 @@ def initialize(client_app)
...
       begin
-        @client_app.create
+        client_app.create
+        #This is currently an unsupported api endpoint, and was derived from the heroku dashboard gui.
+        #Once a tier endpoint solidifies, this functionality should be moved to the 
+        #github_heroku_deployer gem
+        Rails.logger.info "Setting professional tier for #{client_app.name}..."
+        HTTParty.patch("https://api.heroku.com/apps/#{client_app.name}",
+          {
+            body: {name: client_app.name, process_tier: "production"},
```
Comment (thescubageek) [inline lib/client_app_provision_modules.rb]: Nice work

**Comment by thescubageek** [comment]
Have this been tested?

**Comment by variousred** [comment]
@thescubageek yes

**Comment by levibrown** [comment]
with rspec? :P

:shipit:

**Pull request merged**